HW2/03-count.js

```diff
@@ -1,4 +1,4 @@
-// Add your code here
+
 
 //input.addEventListener('keydown', handleKeyDown);
 
```

Comment:
I believe in class Caterina said its best practice to stick to all single quotes or all double quotes for strings, my only recommendation for this file it looks good and I like your use of regular expressions and string interpolation. good job!